kanidm_client/src/lib.rs

```diff
@@ -61,13 +61,15 @@ pub enum ClientError {
 }
 
 #[derive(Debug, Deserialize)]
-struct KanidmClientConfig {
+pub struct KanidmClientConfig {
     uri: Option<String>,
     verify_ca: Option<bool>,
     verify_hostnames: Option<bool>,
     ca_path: Option<String>,
-    // Should we add username/pw later? They could be part of the builder
-    // process ...
+    #[allow(dead_code)]
```

Comment:
I think we just get rid of these two fields for now since we don't use them today.

Reply:
Maybe comment them out? I was going for 'the canonical definition for a Kanidm client config file' to try to keep `pykanidm` and the core def together. Happy either way.

Reply:
We don't use them, so remove them and we can readd if needed